### ISSUE TITLE
Ask for the colour of glass when adding glass to a recycling point 

### ISSUE THREAD

**Issue by Moohan** [streetcomplete/StreetComplete#5120]
As in the title.

In the UK at least, glass is pretty much always separated by colour (brown, green, clear). Usually, but not always there will be a bin for each colour.

When doing the 'recycling point' quest, and selecting glass recycling, users should be given a sub menu to select (multiple can be selected) the colour of glass - Green, Brown, Clear, Mixed. This would be analogous to how plastic recycling currently works.

**Comment by arrival-spring** [comment]
It seems that this has been minimally tagged (less than 0.5% of the time): https://taginfo.openstreetmap.org/search?q=recycling%3Aglass#keys

And there are two competing ways to tag it, `recycling:glass_bottles:colour` or `recycling:glass_green` etc.

That needs to be solved first, with community approval.

Also, what to tag if there is no indication of what colour/s can be recycled there?

**Comment by Moohan** [comment]
Yep, fair enough. Looking through that tag data, there's a real jumble of values for the various tags!

I've not done this before; where/how should consensus be sought, and is there a bot for cleaning up existing tags (recategorising etc.) once (if) there's consensus?

**Comment by arrival-spring** [comment]
Perhaps first ask (informally) on the forums: https://community.openstreetmap.org/

Then/instead make a proposal: https://wiki.openstreetmap.org/wiki/Proposal_process

There's no bot for cleaning up existing tags, but you could make such an edit, following [Automated Edits code of conduct](https://wiki.openstreetmap.org/wiki/Automated_Edits_code_of_conduct)
